src/content/docs/hi/getting-started.mdx

```diff
@@ -2,120 +2,67 @@
-
-📚 Astro के [फ़ाइल-आधारित राउटिंग](/hi/core-concepts/routing/) के बारे में जानें।
-
-*... **संदर्भ** टैब के अंतर्गत हमारे संपूर्ण API के प्रलेखन ढूंढें।*
-
-
-## Astro का विस्तार करें
-
-🧰 अपनी अगली परियोजना [पहले से बनाए गए theme](https://astro.build/themes/) के साथ करे।
-
-🧰 अपनी साइट को आधिकारिक एवं समुदाय [प्लगिन्स एवं अवयवों](https://astro.build/integrations/) के साथ अनुकूलित करें।
-
-🧰 हमारी [साइट प्रदर्शन](https://astro.build/showcase/) पर जाकर प्रेरणा लें।
-
-*... हमारी [एकीकरणों का उपयोग करने के लिए मार्गदर्शिका](/hi/guides/integrations-guide/) देखें*।
-
-
-## हमारी समुदाय के साथ जुड़े
-
-एक सक्रिय, मैत्रीपूर्ण समुदाय के साथ साझा करने एवं उससे सहायता प्राप्त करने के लिए [Astro Discord](https://astro.build/chat) में हमसे जुड़ें!
-
-💬 हमारे `#general` चैनल में hi कहें!
-
-💬 हमारे `#support` चैनल में हमारे सहायता दस्ते से एक प्रश्न पूछें!
-
-💬 आप जिस पर काम कर रहे हैं उसे हमारे `#showcase` चैनल में साझा करें!
-
-
-## Astro में नया क्या है?
-
-[Astro ब्लॉग](https://astro.build/blog/)
-
-[Astro परिवर्तन दैनिकी](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
-
-
-<TranslatorList githubRepo="withastro/docs">
-## Astro Docs का अनुवाद करें
-
-ये प्रलेखन का अनुवाद इन सभी अद्भुत लोगों द्वारा आपके लिए लाए गए हैं। [अनुवाद में हमारी सहायता करें!](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md)
-</TranslatorList>
-
-## Astro में योगदान करें
-
-ये प्रलेखन आपके लिए इन सभी मददगार लोगों द्वारा लाए गए हैं। [GitHub पर हमसे जुड़ें!](https://github.com/withastro/docs)
-
-<ContributorList githubRepo="withastro/docs" />
+import { CardGrid } from '@astrojs/starlight/components'
+import Card from '~/components/Landing/Card.astro'
+import ListCard from '~/components/Landing/ListCard.astro'
+import SplitCard from '~/components/Landing/SplitCard.astro'
+import Discord from '~/components/Landing/Discord.astro'
+
+<CardGrid>
+  <Card title="आप Astro के साथ क्या बनाएंगे?" icon="laptop">
+    ब्लॉग, पोर्टफ़ोलियो, दस्तावेज़, लैंडिंग पेज, SaaS, मार्केटिंग, ईकॉमर्स साइटों और बहुत कुछ के लिए [Astro स्टार्टर थीम](https://astro.build/themes/) का अन्वेषण करें!
+  </Card>
+
+  <Card title="एक निर्देशित भ्रमण करें" icon="star">
+    बुनियादी बातें सीखने और अपनी पहली Astro साइट बनाने के लिए हमारे परिचयात्मक [ब्लॉग ट्यूटोरियल](/hi/tutorial/0-introduction/) को पूरा करें।
+  </Card>
+
+  <SplitCard title="एक नयी परियोजना शुरू करें" icon="rocket">
+    ```sh
+    # npm के साथ एक नयी परियोजना बनाएं
+    npm create astro@latest
+    ```
+
+    हमारे [इंस्टॉलेशन मार्गदर्शक](/hi/install/auto/) में हमारे CLI विज़ार्ड का उपयोग करके Astro को इंस्टॉल करने, मौजूदा Astro GitHub रिपॉजिटरी से एक नया परियोजना बनाने और Astro को नियमावली रूप से इंस्टॉल करने के लिए चरण-दर-चरण निर्देश हैं।
+  </SplitCard>
+
+  <ListCard title="सीखे " icon="open-book">
+    - [Astro की मुख्य विशेषताएं](/hi/concepts/why-astro/)
+    - [द्वीप वास्तुकला](/hi/concepts/islands/)
+    - [Astro अवयव](/hi/core-concepts/astro-components/)
```

Comment:
Keep it in English.

Reply:
Can I know why you want it in English? The question is applicable to all other suggestions

Reply:
Because some terms are not supposed to be translated, like a developer would know about components but 'अवयव' doesn't make much sense, not only that, but no one practices this level of pure Hindi in their day-to-day conversation.